src/controllers/update/update.filing.date.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    return res.render(config.UPDATE_FILING_DATE_PAGE, {
+      backLinkUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
+      templateName: config.UPDATE_FILING_DATE_PAGE
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
```

Comment:
This is the first page where the "Save" of the "Save and Resume" journey in ROE Update.
So here we need to "Open" The Transaction.
Remove the "Open" Transaction where it is now in CYA Page
Save the TransactionId in the session
This page also needs to have the "save and continue" button

Reply:
@bantao-ch , isn't that supposed to be part of https://companieshouse.atlassian.net/browse/UAR-333, as it explicitly mentions opening and closing the transaction

Reply:
See Tech note number 2 in UAR-333 it references the open transaction back to this one.
UAR-333 is about adding the save-and-continue button to the rest of the savable pages.